package.json

```diff
@@ -11,7 +11,7 @@
     "email": "contact@os-scar.com"
   },
   "scripts": {
-    "postinstall": "husky install",
+    "postinstall": "husky install && yarn playwright install",
```

Comment:
playwright install --with-deps chromium
maybe this will be more accurate.
https://playwright.dev/docs/browsers

Reply:
@baruchiro

Reply:
```suggestion
    "postinstall": "husky install && yarn playwright install --with-deps chromium",
```